draft-ietf-tls-dtls13.md

```diff
@@ -1532,24 +1532,44 @@ The configuration of timer settings varies with implementations and certain
 deployment environments require timer value adjustments. Mishandling
 of the timer can lead to serious congestion problems, for example if
 many instances of a DTLS time out early and retransmit too quickly on
-a congested link.  Implementations SHOULD use an initial timer value
-of 100 msec (the minimum defined in RFC 6298 {{RFC6298}}) and double
-the value at each retransmission, up to no less than 60 seconds
-(the RFC 6298 maximum). Application specific profiles, such as those
-used for the Internet of Things environment, may recommend longer
-timer values. Note that a 100 msec timer is recommended
-rather than the 3-second RFC 6298 default in order to improve latency
-for time-sensitive applications.  Because DTLS only uses
-retransmission for handshake and not dataflow, the effect on
-congestion should be minimal.
+a congested link.
+
+Unless implementations have extra information about the round trip time, 
+implementations SHOULD use an initial timer value of 1000 ms and double 
+the value at each retransmission, up to no less than 60 seconds (the 
+RFC 6298 {{RFC6298}} maximum). Application specific profiles, MAY 
+recommend shorter or longer timer values. For instance:
+
+* Profiles for specific deployment environments, such as in low-power, 
+  multi-hop mesh network as used in some Internet of Things (IoT) networks, 
+  MAY specify longer timeouts. See {{?I-D.ietf-uta-tls13-iot-profile}} for 
+  more information about one such DTLS 1.3 IoT profile.
+
+* Real-time protocols MAY specify shorter timeouts. It is RECOMMENDED
+  that for DTLS-SRTP {{?RFC5764}}, a default timeout of
+  400ms be used; because customer experience degrades with one-way latencies
+  of greater than 200ms, real-time deployments are less likely
+  to have long latencies. 
+
+In settings where there is external information (for instance from an ICE {{?RFC8445}}
+handshake, or from previous connections to the same server)
+about the RTT, implementations SHOULD use that to set the retransmit
+time. 
```

Comment:
There should probably be some text about how to "use that to set the retransmit time", i.e., simply reusing that value is not what TCP does. I think Linux does something like `RTO = max(SRTT + 4 * RTTVAR, 1000ms)`. [QUIC does something similar](https://www.ietf.org/archive/id/draft-ietf-quic-recovery-34.html#section-6.2.1).

Reply:
Sure. I'll make this 1.5RTT as suggested by Allman and used elsewhere in this doc,